test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -141,3 +141,36 @@ def count_node_state(self, state='UP  B'):
             if stat['State'] == state:
                 count += 1
         return count
+
+    def get_numthreads(self, nid):
+        """
+        Returns the number of hyperthread for the given node
+        """
+        options = '-N %d -n -f "nid,c/cu"' % int(nid)
+        hostname = socket.gethostname()
+        platform = self.du.get_platform(hostname)
+        apstat_env = os.environ
+        apstat_cmd = "apstat"
+        if 'cray' not in platform:
+            return NULL
+        if 'craysim' in platform:
+            lib_path = '$LD_LIBRARY_PATH:/opt/alps/tester/usr/lib/'
+            apstat_env['LD_LIBRARY_PATH'] = lib_path
+            apstat_env['ALPS_CONFIG_FILE'] = '/opt/alps/tester/alps.conf'
+            apstat_env['apsched_sharedDir'] = '/opt/alps/tester/'
+            apstat_cmd = "/opt/alps/tester/usr/bin/apstat -d ."
```

Comment:
In the Cray simulator I'm using the apstat help does not show the -d option.  This looks ok since it is being used internally by the Cray simulator in the environment and boot scripts.

Reply:
Okay.  So as I understand it, this is just a comment, and there's no change that you are requesting.  Is my understanding correct?

Reply:
Correct.